packages/editor-ui/src/api/workflow-webhooks.ts

```diff
@@ -7,25 +7,25 @@ const CONTACT_EMAIL_SUBMISSION_ENDPOINT = '/accounts/onboarding';
 
 export async function fetchNextOnboardingPrompt(
 	instanceId: string,
-	currentUser: IUser,
+	currentUser: IUser | null,
 ): Promise<IOnboardingCallPrompt> {
 	return await get(N8N_API_BASE_URL, ONBOARDING_PROMPTS_ENDPOINT, {
 		instance_id: instanceId,
-		user_id: `${instanceId}#${currentUser.id}`,
-		is_owner: currentUser.isOwner,
-		survey_results: currentUser.personalizationAnswers,
+		user_id: `${instanceId}#${currentUser?.id}`,
+		is_owner: currentUser?.isOwner || false,
```

Comment:
Prefer `??` when coalescing

```suggestion
		is_owner: currentUser?.isOwner ?? false,
```